python/custom/templates/PerLineMcdc/tables.html.template

```diff
@@ -0,0 +1,58 @@
+{#- Macro definitions -#}
```

Comment:
Remove this for now; we'll add it it back in a later PR

Reply:
Removed in b47303ebd7cd9ac952b379fb59fcb96cd0b4c57d